docs/source/json-api/index.rst

```diff
@@ -659,6 +660,7 @@ HTTP Request
 Where:
 
 - ``templateId`` -- contract template or interface identifier, same as in :ref:`create request <create-request>`,
+- ``choiceInterfaceId`` -- *optional* template or interface that defines the choice, same format as ``templateId``,
```

Comment:
The documentation should include an explanation of why this is optional and what are the consequences of defining or _not_ defining this.

Reply:
How does 0eeb53190768ff1d43307362a194ff2904b25e63 look?

Reply:
Looks good, thanks for going in depth.